examples/basic/kernel/basic-kernel.md

```diff
@@ -0,0 +1,11 @@
+# basic-kernel
+
+A minimal kernel to showcase the usage of bootloader.
+
+The kernel:
+- Initalizes a serial port on [COM1](https://wiki.osdev.org/Serial_Ports#Port_Addresses)
+- Dumps the [`boot_info`](/api/src/info.rs) it recevied from `bootloader`
+- Prints a message
+- Exits QEMU with a custom exit code via the `isa-debug-exit` device
+
+Check https://os.phil-opp.com/ for further information on how to write an OS in Rust. 
```

Comment:
```suggestion
```
That tutorial is already very widely known. I'd guess that most people are already familiar with that one and if they aren't I think we also want people to give a chance to stumble upon other tutorials that have been written since.